tools/SetupFlow/DevHome.SetupFlow/Views/SummaryView.xaml

```diff
@@ -113,72 +115,237 @@
                         x:Uid="ms-resource:///DevHome.SetupFlow/Resources/SummaryPage_HeaderWithErrors"/>
                 </Grid>
 
-                <Grid Grid.Row="2" ColumnSpacing="25" Visibility="{x:Bind ViewModel.ShowConfigurationUnitResults, Mode=OneWay}">
+                <Grid Grid.Row="2" ColumnSpacing="50" Visibility="{x:Bind ViewModel.ShowConfigurationUnitResults, Mode=OneWay}">

```

Comment:
Did you have a chance to review these changes for accessibility issues?

Reply:
This is the same value as for local case below. I assume the local setup was reviewed when it was introduced.
This whole change duplicates most of the XAML used for the local setup with replacement of variables that needed for target setup.

Reply:
I would recommend changing the text size from Settings -> Accessibility and checking once though.